src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -21,14 +24,17 @@ public class IexService {
   @NonNull
   private IexClient iexClient;
 
+  @Value("${spring.datasource.token}")
+  private String token;
+
```

Comment:
Looks like you are getting the property here and as the default value in the IexClient. This is duplicative, I think that the IexClient can use the defaulting functionality to retrieve this value rather than passing it from the IexService.